src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2317,6 +2321,7 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
             workspace.FileName = string.IsNullOrEmpty(filePath) ? "" : filePath;
             workspace.FromJsonGraphId = string.IsNullOrEmpty(filePath) ? WorkspaceModel.ComputeGraphIdFromJson(fileContents) : "";
             workspace.ScaleFactor = dynamoPreferences.ScaleFactor;
+            workspace.EnableLegacyPolyCurveBehavior = dynamoPreferences.EnableLegacyPolyCurveBehavior;
```

Comment:
so the preferences override the workspace data?

Reply:
No, it should be the other way around.

Reply:
If there is no workspace data related to this flag, the one in preferences is used otherwise the one in the workspace.

Reply:
We do not expose internals to the libg assemblies ? maybe there’s a reason for that…

Reply:
@pinzart90 are you suggesting I make the new property that's exposed to LibG internal and make the internals visible to the LibG assembly?